mscore/inspector/inspector.cpp

```diff
@@ -185,9 +186,19 @@ void Inspector::update(Score* s)
                         sameTypes = false;
                         break;
                         }
+                      // an articulation and an ornament
+                  if ((ee->isArticulation() && toArticulation(ee)->isOrnament() != toArticulation(element())->isOrnament()) ||
+                      // a slur and a tie
+                      (ee->isSlurTieSegment() && toSlurTieSegment(ee)->accessibleInfo() != toSlurTieSegment(element())->accessibleInfo()) ||
```

Comment:
We probably shouldn't rely on `accessibleInfo()` to determine the element's subtype. `accessibleInfo()` is supposed to be a human-readable textual representation of the relevant information, and it may eventually happen to include information about location or voice number or other context for this element.

Reply:
OK, will change it.

Reply:
It is a bit of a mean trick for sure

Reply:
"Back then", `isSlurTieSegment()` is available but `isSlurSegment()` isn't??

Reply:
Maybe it was, but I was using the same method, `accessibleInfo()`, in several places